### DISCUSSION TITLE
Official solid-js support

### DISCUSSION THREAD

**Discussion by hackwaly** [pmndrs/jotai#1683] [Ideas]
What the big thing missed in solid-js is that it can't preserve component state when doing HMR. This is what jotai be good at. Just wanna see jotai ❤️ solid-js.

Here's an unoffical version https://github.com/wobsoriano/solid-jotai.

**Comment by dai-shi** [comment]
I didn't know that jotai is good at HMR. Do you mean the babel plugin?
The `solid-jotai` package is actually pretty good. Have you tried it?

**Reply by land-cap** [reply]
Hi @dai-shi first of all huge thanks for such a great library. I can't go back to anything else now 😃

I think I am among the many who really like Solid but the only thing holding them back from switching from React is that its buit-in signals don't provide all the great features Jotai has and while there is this wrapper you mention I think everyone is still waiting for an official solution "blessed" by the core maintainer to be sure it's the best integration and production ready🙂. It would be highly appreciated if we had that even if it's just the same package but co-owned or by pmdrs.

**Reply by dai-shi** [reply]
Unfortunately, we are not experts with solidjs. That said, when I reviewed the `solid-jotai` code in the past, it was organized well, and I didn't see anything wrong from the Jotai perspective.
If @wobsoriano is interested in moving `solid-jotai` library to https://github.com/jotaijs, I'm happy to invite them.

**Reply by wobsoriano** [reply]
> Unfortunately, we are not experts with solidjs. That said, when I reviewed the `solid-jotai` code in the past, it was organized well, and I didn't see anything wrong from the Jotai perspective.
> 
> If @wobsoriano is interested in moving `solid-jotai` library to https://github.com/jotaijs, I'm happy to invite them.

Let's do it 🙏

**Reply by dai-shi** [reply]
Sent an invitation.

**Reply by wobsoriano** [reply]
Thanks @dai-shi! Do we want to just move it into a new repo or inside [jotai](https://github.com/pmndrs/jotai) repo?

**Reply by dai-shi** [reply]
Transfer your repo to the jotaijs org. You are still the admin.
